File: AY25-26-6th Sem - ISE 1/Seventeen.java
package Day3;

import java.util.Scanner;

public class Seventeen {

	public static void main(String[] args) {
	Scanner scanner=new Scanner(System.in);
	int num=scanner.nextInt();
	int count=0,num2=num,digit,sum=0;
	while(num!=0)
	{
		count++;
		num=num/10;
	}
num=num2;
while(num!=0)
{
	digit=num%10;
	sum=sum+(int)Math.pow(digit,count);
	num=num/10;
	count--;
}
if(sum==num2)
{
	System.out.println("Disarium");
	
}
else {
	System.out.println("Not disarium");
}
	}

}

File: AY25-26-6th Sem - EEE 1/Seventeen.c
//153 =1^3+5^3+3^3


#include<stdio.h>
#include<math.h>
int main()
{
    int num,count=0,sum=0,num2,digit;
    scanf("%d",&num);
    num2=num;
    while(num!=0)
    {
        num=num/10;
        count++;
    }
    num=num2;
    while(num!=0)
    {
        digit=num%10;
        sum=sum+pow(digit,count); 
        num=num/10;
    }
    if(num2==sum)
    {
        printf("\nAmstrong");
    }
    else{
        printf("\nNot amstrong ");
    }
    return 0;
}